status-bar/android/build.gradle

```
@@ -51,6 +51,7 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
+    implementation "androidx.core:core:1.7.0"
```

```suggestion
    implementation "androidx.core:core:$androidxCoreVersion"
```
use the androidxCoreVersion variable instead of hardcoded 1.7.0

don't need to be documented since it's one of the default variables included in variables.gradle, but need to be declared in the ext section on top of the file